tap_chargebee/client.py

```diff
@@ -70,19 +179,9 @@ def make_request(self, url, method, params=None, body=None):
             params=self.get_params(params),
             json=body)
 
-        try:
-            response_json = response.json()
-        except simplejson.scanner.JSONDecodeError:
-            # Formatted error message for json decoder error
-            response_json = {
-                "message": "Did not get response from the server due to an unknown error.",
-                "http_status_code": response.status_code
-            }
-
-        if response.status_code == 429:
-            raise Server429Error()
+        if response.status_code != 200:
+            raise_for_error(response)
 
-        if response.status_code >= 400:
-            raise Server4xxError(response_json)
+        response_json = response.json()
```

Comment:
We directly return `response.json()`

Reply:
Yes, updated code to return response.json() directly.